### ISSUE TITLE
Check this out

### ISSUE THREAD

**Issue by Aotoge** [mrsaicharan1/Algorithm-Design-Report1#2]
![screenshot from 2018-01-26 14 10 46](https://user-images.githubusercontent.com/22166269/35431554-e4bffbb2-02a2-11e8-9d95-0464d11ae1d1.png)

^Not exactly correct :cry:

**Comment by mrsaicharan1** [comment]
Ohhhhh

**Comment by mrsaicharan1** [comment]
%.2f would do the trick I guess

**Comment by Aotoge** [comment]
Not really if you do 123.456 it will round the result to 0.46

**Comment by mrsaicharan1** [comment]
We'll narrow down the precision to 3 digits. Let's just mention it in the assumptions part of the report.

**Comment by Aotoge** [comment]
I dont think that would be needed. We have to separate out each digit, as it is

**Comment by mrsaicharan1** [comment]
Alright, we'll separate it.
